code/__DEFINES/~skyrat_defines/spacepod_helpers.dm

```diff
@@ -0,0 +1,8 @@
+#define isspacepod(A) (istype(A, /obj/spacepod))
+#define isspacepodequipment(A) (istype(A, /obj/item/spacepod_equipment))
+
+/// Sends a signal when the client detects mouse movement
+/client/MouseMove(atom/object, atom/location, control, params)
+	. = ..()
+	if(mob)
+		SEND_SIGNAL(mob, COMSIG_MOB_CLIENT_MOUSE_MOVE, object, location, control, params)
```

Comment:
I am incredibly, incredibly sus of this code
to quote MSO
![image](https://user-images.githubusercontent.com/41448081/231355911-b70617ae-f016-479e-9b9c-4fe5b970fb04.png)

![image](https://user-images.githubusercontent.com/41448081/231356095-bd7db63e-f6d1-4947-93a9-b6f340a02eff.png)



Bare minimum, I want this thoroughly profiled on live before merge

Reply:
as a note, there is a cheap way to do this

look at how the Kinesis module works for MODSuits(2021) (by fikou)

Reply:
Its on the to do